_static/js/search.js

```diff
@@ -5,29 +5,34 @@ function hideEverything() {
     $("#helpmessage").hide();
 }
 
+
 window.addEventListener("load", function() {
     $('.ui.button').click(function() {
         $.getJSON("/api/check/" + encodeURIComponent($("input").val()), function(result) {
             if (result.result == 'verified') {
                 hideEverything();
+                $("#verifiedmessage").html($("input").val() + ' is a verified domain. You can trust the contents.');
```

Comment:
We need to add some sanitation here around `$("input").val()`. With the current implementation we can execute rouge JS - although not damaging in this application, we still need to sanitise.